<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="bg" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>450</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#bg{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(85, 255, 255, 255), stop:1 rgba(255, 255, 255, 255))}
</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>450</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 28pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Current Traffic Statistics</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="PlotWidget" name="graphWidget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>430</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
border-radius:20px;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="daysbox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <item>
      <property name="text">
       <string>Today</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 day ago</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 days ago</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3 days ago</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 days ago</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5 days ago</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6 days ago</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="areabox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <item>
      <property name="text">
       <string>--Choose an area--</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Jalan Templer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Jalan Gasing</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Jalan PJ</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="backb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
